src/main/scala/com/github/unchama/seichiassist/subsystems/gacha/bukkit/actions/BukkitGrantGachaPrize.scala

```diff
@@ -28,21 +28,23 @@ class BukkitGrantGachaPrize[F[_]: Sync: OnMinecraftServerThread](
       }
     }
 
+  import cats.implicits._
+
   override def insertIntoPlayerInventoryOrDrop(
     prize: GachaPrize[ItemStack]
   ): Kleisli[F, Player, GrantState] =
     Kleisli { player =>
-      Sync[F].delay {
-        val newItemStack = prize.materializeWithOwnerSignature(player.getName)
-        if (!InventoryOperations.isPlayerInventoryFull(player)) {
-          InventoryOperations.addItem(player, newItemStack)
-          GrantState.AddedInventory
-        } else {
-          InventoryOperations.dropItem(player, newItemStack)
-          GrantState.Dropped
-        }
-      }
-    }
+      val newItemStack = prize.materializeWithOwnerSignature(player.getName)
+      for {
+        isInventoryFull <- Sync[F].delay(InventoryOperations.isPlayerInventoryFull(player))
+        _ <-
+          if (isInventoryFull) {
+            Sync[F].delay(InventoryOperations.addItem(player, newItemStack))
+          } else {
+            InventoryOperations.grantItemStacksEffect(newItemStack).apply(player)
+          }
```

Comment:
```suggestion
        _ <-if (isInventoryFull) {
            Sync[F].delay(InventoryOperations.addItem(player, newItemStack))
          } else {
            InventoryOperations.grantItemStacksEffect(newItemStack).apply(player)
          }
```

scalafix等でこうなったのであればこのままで構いませんが、`if`式の部分を上に書きませんか？

Reply:
IDEコードスタイル設定の「保存時に整形」を有効にしていると上記の様に改行されるようです。scalafmt等では発生しません。
保存時に整形をオフにしたら改行されなくなりました。
個人的には自動整形オンのほうが良いのですが、見苦しいのであれば自動整形はオフにします。

Reply:
であればそのままでも大丈夫です。

Reply:
scalafmtでも改行されることを確認しました。そのままで行きたいと思います。